pkg/controller/migmigration/restore.go

```diff
@@ -629,9 +630,13 @@ func (t *Task) deleteStalePVRsOnCluster(cluster *migapi.MigCluster) (int, error)
 
 // Delete namespace and cluster-scoped resources on dest cluster
 func (t *Task) deleteMigrated() error {
-	// TODO: delete 'deployer' Pods associated with DCs even if phase=completed
+	// Delete 'deployer' and 'hooks' Pods that DeploymentConfig leaves behind upon DC deletion.
```

Comment:
If explicit hook pod deletion isn't needed here, it might be better to remove the comment about it.

Reply:
Will update, thanks.